cmd/git-sync/main.go

```diff
@@ -353,7 +354,7 @@ func setFlagDefaults() {
 // it is deadlocked.
 func sleepForever() {
 	c := make(chan os.Signal, 1)
-	signal.Notify(c, os.Interrupt, os.Kill)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```

Comment:
SIGKILL cannot be trapped

Reply:
That was the point - it's not actually trying to do anything, just trick Go's runtime into thinking it is not hung.  This leaves "real" signals to be handled elsewhere if needed, which seems plausible for SIGINT.

I propose we nix this - I am thinking about changing this functionality anyway, so it won't matter long term.

Reply:
Ok, thanks for the explanation. Reverted.